### DISCUSSION TITLE
use wrfout files to run HICAR

### DISCUSSION THREAD

**Discussion by wwwliye** [NCAR/icar#192] [Q&A]
Hi , recently I'm trying to use wrfout files to run HICAR. I want to know if I want to input wrfout, firstly should I use helpers/wrf/wrf2icar.py to convert wrfout, secondly I put the file to HICAR ,but there are some questions.. so to input wrfout, what is the true steps? thank you for your help

**Comment by gutmann** [comment]
I would recommend looking at the wrf2icar script.  It likely needs some modifications as it hasn't been touched in a while (yikes, 7 years!), but if would give you an idea of where to begin. I have not used the HICAR variant much, @d-reynolds would be a better source of information for that, though I believe most or all of the input variables are identical.

**Comment by gutmann** [comment]
Major things that should probably be modified: 
WRF wind fields are grid-relative and must be rotated to be earth relative before input to ICAR. 
This requires destaggering the wind field, that's good, just leave it on the destaggered grid. 
P and PB must be combined, as with PH and PHB. If you use PH in native units, set the "z_is_geopotential" namelist option to true. 
If T is in WRF's odd [K - 300] units, either set the temperature offset in the namelist (if that still works, I haven't used that in ages either) or just change the units to K before use.
